apache/init.sls

```diff
@@ -10,6 +10,7 @@ apache:
     - name: {{ apache.user }}
     - gid: {{ apache.group }}
     - system: True
+  {% if apache.manage_service_states in (True, 'True', None, '') %}
```

Comment:
Please explicitly use a default value here. Since we're not using `map.jinja`, we're not in the `apache:lookup` realm of Pillar, and we ought to use the Pillar directly:

```sls
{% if salt['pillar.get']('apache:manage_service_states', True) %}
```